tasks/sync-granule/index.js

```diff
@@ -69,6 +69,7 @@ exports.syncGranule = function syncGranule(event) {
     buckets,
     collection,
     provider,
+    fileStagingDir,
```

Comment:
There is a default here, right? It might help to put a comment in the config.json what the default is and maybe in here because upon first glance it looks like it should be required.

Reply:
:+1: